v2/go.mod

```diff
@@ -10,12 +12,12 @@ require (
 	github.com/ipfs/go-datastore v0.6.1-0.20230901172804-1caa2449ed7c
 	github.com/ipfs/go-ds-leveldb v0.5.0
 	github.com/ipfs/go-log/v2 v2.5.1
-	github.com/libp2p/go-libp2p v0.30.0
+	github.com/libp2p/go-libp2p v0.29.1-0.20230817072656-d2398ee4f251
```

Comment:
Why this downgrade? Is this a result from the `replace` directive above?

Reply:
Restored it, related to the above where I was iterating towards a combination of libp2p dependencies that built together